### PR TITLE
pdesjardins/DOC-2358

### PR DESCRIPTION
**Pull request by pdesjardins**
## [DOC-2358](https://openedx.atlassian.net/browse/DOC-2358)

DOC-2358 reports that a user was confused by the empty "desktop_webm URL" column in the available encodings report for video uploads. I added more information to the explanation of that column in the Reporting Video Statuses section.
### Reviewers

Possible roles follow. PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [x] Subject matter expert: cdh@edx.org
- [x] Doc team review (sanity check/copy edit/dev edit): @catong @srpearce @lamagnifica 
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [x] Squash commits

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @pdesjardins! I've created [OSPR-969](https://openedx.atlassian.net/browse/OSPR-969) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).
We can't start reviewing your pull request until you've   submitted a [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)   or indicated your institutional affiliation   and   added yourself to the [AUTHORS](https://github.com/edx/edx-documentation/blob/master/AUTHORS) file . Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

File: en_us/course_authors/source/video/video_uploads.rst [hunk 1]
```diff
@@ -266,28 +266,26 @@ The CSV file reports data in these columns for all of the files you upload.
...
   format. This file is delivered to learners who do not have access to YouTube
-  and view course videos with an mp4 player.
+  and view course videos with mp4 players.
 
 * **desktop_webm URL**: The AWS location of a 720p resolution video file in
   webm format. This file is delivered to learners who do not have access to
-  YouTube and view course videos with a webm player.
+  YouTube and view course videos with webm players.
+
+  The encoding and hosting process no longer creates webm versions of the video files that you upload. Modern web browsers do not require the webm format. The CSV file includes the **desktop_webm URL** column to show the webm URLs for videos uploaded before this change. When you upload a new video, the column will remain empty, even after the encoding and hosting process is complete.
```
Comment (srpearce) [inline en_us/course_authors/source/video/video_uploads.rst]: Line wrap - and make sure that the entire paragraph is indented (flush with the first line) when the lines wrap.

File: en_us/course_authors/source/video/video_uploads.rst [hunk 2]
```diff
@@ -249,7 +249,7 @@ file.
 
 #. Use a spreadsheet application or text editor to open the CSV file.
 
-The CSV file reports data in these columns for all of the files you upload.
+The CSV file includes the following columns of information.
```
Comment (srpearce) [inline en_us/course_authors/source/video/video_uploads.rst]: Strike "of information"

File: en_us/course_authors/source/video/video_uploads.rst [hunk 3]
```diff
@@ -237,11 +237,11 @@ problem. Inform your edX Partner Manager if these statuses appear.
 Reporting Video Statuses
 ================================
 
-To report the status of the encoding and hosting process of every video file
-that you upload, as well as the assigned video IDs, you can download a CSV
-file.
+View detailed information about the video files that you upload in the available encodings report. The available encodings report includes the status of the encoding and hosting process for each video file that you upload, the identifier for the video, and the URLs for each encoding format. The available encodings report is a Comma Separated Values (CSV) file that you can view in a spreadsheet application or text editor.
```
Comment (srpearce) [inline en_us/course_authors/source/video/video_uploads.rst]: Global comment: Press COMMAND+OPTION+Q to wrap lines. (If that doesn't work, make sure you've installed all the Sublime packages that are listed on the Documentation Tools page.)

Decap "comma-separated values" and change (CSV) to (.csv)*

*We aren't entirely consistent with this in our docs, but of the 95 instances of "csv" or "CSV", 75 are ".csv"—so I'd go with that.

File: en_us/course_authors/source/video/video_uploads.rst [hunk 4]
```diff
@@ -8,14 +8,14 @@ After a video administrator works with the edX media team to complete
 :ref:`preliminary setup<Video Getting Started>` for the entire institution,
```
Comment (lamagnifica) [inline en_us/course_authors/source/video/video_uploads.rst:8]: @pdesjardins FYI, DB, who you met at lunch, had us all add a check for extraneous spaces and lines to Sublime recently. We're still seeing the fall out from doing so in our PRs....

File: en_us/course_authors/source/video/video_uploads.rst (hunk 3 above)
Comment (lamagnifica) [inline en_us/course_authors/source/video/video_uploads.rst]: FYI you might need to add a package to Sublime to get the line wrap command to work. See https://openedx.atlassian.net/wiki/display/DOC/Documentation+Tools.

We are continually trying to make the docs consistent, and get them to comply with the style guide (first version published internally a year ago, but editing for consistency hasn't been prioritized very highly). But to the extent possible, please do try to follow the guide and bring the files that you touch up to date with it!

File: en_us/course_authors/source/video/video_uploads.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/course_authors/source/video/video_uploads.rst]: and that CSV --> .csv change can be a global replace in this rst

File: en_us/course_authors/source/video/video_uploads.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/course_authors/source/video/video_uploads.rst]: to reduce the use of future tense, I'd suggest "the column will remain empty, " --> "this column remains empty, "

File: en_us/course_authors/source/video/video_uploads.rst [hunk 5]
```diff
@@ -266,28 +266,26 @@ The CSV file reports data in these columns for all of the files you upload.
...
-  YouTube and view course videos with a webm player.
+  YouTube and view course videos with webm players.
+
+  The encoding and hosting process no longer creates webm versions of the video files that you upload. Modern web browsers do not require the webm format. The CSV file includes the **desktop_webm URL** column to show the webm URLs for videos uploaded before this change. When you upload a new video, the column will remain empty, even after the encoding and hosting process is complete.
 
 * **mobile_low URL**: The AWS location of a 320p resolution video file. This
-  file is delivered to learners who download and view course videos on a
-  mobile device.
+  file is delivered to learners who download and view course videos on
+  mobile devices.
```
Comment (lamagnifica) [inline en_us/course_authors/source/video/video_uploads.rst:297]: thank you for fixing the agreement throughout!

**Comment by lamagnifica** [comment]
great. a few suggestions, otherwise :+1:

**Pull request merged**